### PR TITLE
go.mod: update starlark dependency

### LINKED ISSUE #2907: dlv tries to allocate a 4GiB memory chunk and failed with ENOMEM
1. What version of Delve are you using (`dlv version`)?
Should be the latest release
2. What version of Go are you using? (`go version`)?
```
go version go1.16.7 linux/amd64
```
3. What operating system and processor architecture are you using?
OS:
```
NAME="openSUSE Leap"
VERSION="15.3"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.3"
PRETTY_NAME="openSUSE Leap 15.3"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.3"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```
Processor:
```
processor       : 0
vendor_id       : GenuineIntel
cpu family      : 6
model           : 63
model name      : Intel(R) Xeon(R) CPU E5-2698 v3 @ 2.30GHz
stepping        : 2
microcode       : 0x46
cpu MHz         : 1906.780
cache size      : 40960 KB
physical id     : 0
siblings        : 32
core id         : 0
cpu cores       : 16
apicid          : 0
initial apicid  : 0
fpu             : yes
fpu_exception   : yes
cpuid level     : 15
wp              : yes
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm cpuid_fault epb invpcid_single pti ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid cqm xsaveopt cqm_llc cqm_occup_llc dtherm ida arat pln pts md_clear flush_l1d
bugs            : cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit
bogomips        : 4594.97
clflush size    : 64
cache_alignment : 64
address sizes   : 46 bits physical, 48 bits virtual
power management:

```
4. What did you do?
run `dlv version`
5. What did you expect to see?
Success output with version
6. What did you see instead?
An error `2022/02/14 06:54:01 mmap: cannot allocate memory`
The [strace log](https://pastebin.com/k056mUDB) shows that dlv is trying to perform the following syscall which failed:
```
mmap(NULL, 4294967296, PROT_READ, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = -1 ENOMEM (Cannot allocate memory)
```

### PR DESCRIPTION
**Pull request by aarzilli**
Fixes issue where Delve fails to start if starlark can not allocate a
large amount of memory at startup.

Fixes #2907

### CODE REVIEW

File: pkg/proc/native/proc_freebsd.go [hunk 1]
```diff
@@ -20,6 +20,12 @@ import (
 	isatty "github.com/mattn/go-isatty"
 )
 
+const (
```
Comment (derekparker) [inline pkg/proc/native/proc_freebsd.go:23]: Why copy these constants over?

Comment (aarzilli) [reply pkg/proc/native/proc_freebsd.go:23]: They were removed from x/sys/unix at some point.

Comment (derekparker) [reply pkg/proc/native/proc_freebsd.go:23]: Ack.

**Review by derekparker — APPROVED** [review]
LGTM

**Pull request merged**